edweiss-firebase/functions/src/functions/todolist/deleteTodo.ts

```diff
@@ -0,0 +1,19 @@
+import Todolist from 'model/todo';
+import { onAuthentifiedCall } from 'utils/firebase';
+import { CollectionOf, getDocumentRef } from 'utils/firestore';
+import { fail, ok } from 'utils/status';
+
+import Functions = Todolist.Functions;
+
+export const deleteTodo = onAuthentifiedCall(Functions.deleteTodo, async (userId, args) => {
```

Comment:
Catches any errors that occur during the deletion process and returns a "firebase_error" status..

Reply:
I will refine it later, but that a great point for greating bulletproof software ...